packages/admin-ui/styles/components/button.less

```diff
@@ -0,0 +1,6 @@
+
+.adm-btn {
+  &-gray {
+    .button-variant-primary(@gray-8; @gray-5; @gray-6; @gray-7);
```

Comment:
As discussion before, we adjust the button colors to:
- bg: gray-6
- bg-hover: gray-5
- bg-active: gray-7
- color: gray-9